packages/deployment/app/cloudformation.template.yml

```diff
@@ -518,18 +484,6 @@ Resources:
   {{/each}}
 {{/each}}
 
-## Generic lambda permission for custom rules
-## created in the dashboard
-  GenericLambdaPermission:
```

Comment:
This had to be moved into the API template because it refers to a lambda, ScheduleSF, that is now generated within the API template

Reply:
I think this is fine - it sounds to me like this resource can be considered part of the API deployment. This may be a dumb question, but just to be sure, if we _didn't_ deploy the API stack (and so didn't have this `GenericLambdaPermission`, would that cause issues in other areas?